TypeScript build error: Return type of exported function has or is using name 'BaseWebhookEvent' from external module "/Users/gregor/Projects/octokit/app.js/node_modules/@octokit/webhooks/dist-types/types" but cannot be named.

Occurred here after upgrading to `@octokit/webhooks` v8:
https://github.com/octokit/app.js/runs/1858152718?check_suite_focus=true#step:5:14

```
node_modules/typescript/bin/tsc --outDir pkg/dist-src/ -d --declarationDir pkg/dist-types/ --project tsconfig.json --target es2020 --module esnext --noEmit false --sourceMap false
src/webhooks.ts:8:17 - error TS4058: Return type of exported function has or is using name 'BaseWebhookEvent' from external module "node_modules/@octokit/webhooks/dist-types/types" but cannot be named.

8 export function webhooks(
                  ~~~~~~~~
```

@G-Rath @wolfy1339 @oscard0m does that error ring a bell?

--- a/src/webhooks.ts
+++ b/src/webhooks.ts
@@ -8,7 +8,7 @@ import { Options } from "./types";
 export function webhooks(
   appOctokit: Octokit,
   options: Required<Options>["webhooks"]
-) {
+): Webhooks {
   return new Webhooks({
     secret: options.secret,
     path: "/api/github/webhooks",

--- a/src/webhooks.ts
+++ b/src/webhooks.ts
@@ -1,14 +1,14 @@
 import { Octokit } from "@octokit/core";
 import { createAppAuth } from "@octokit/auth-app";
 import { createUnauthenticatedAuth } from "@octokit/auth-unauthenticated";
-import { Webhooks } from "@octokit/webhooks";
+import { Webhooks, EmitterWebhookEvent } from "@octokit/webhooks";
 
 import { Options } from "./types";
 
 export function webhooks(
   appOctokit: Octokit,
   options: Required<Options>["webhooks"]
-): Webhooks {
+): Webhooks<EmitterWebhookEvent, EmitterWebhookEvent & { octokit: Octokit }> {
   return new Webhooks({
     secret: options.secret,
     path: "/api/github/webhooks",

--- a/src/webhooks.ts
+++ b/src/webhooks.ts
@@ -8,7 +8,7 @@ import { Options } from "./types";
 export function webhooks(
   appOctokit: Octokit,
   options: Required<Options>["webhooks"]
-): Webhooks<EmitterWebhookEvent, EmitterWebhookEvent & { octokit: Octokit }> {
+): Webhooks<EmitterWebhookEvent & { octokit: Octokit }> {
   return new Webhooks({
     secret: options.secret,
     path: "/api/github/webhooks",

